test/helper.rb

```diff
@@ -10,6 +10,7 @@
 require "phenix"
 
 ENV["RAILS_ENV"] = "test"
+TEST_ADAPTER_MYSQL = ENV.fetch("TEST_ADAPTER_MYSQL") { :mysql2 }.to_sym
```

Comment:
Do we need the `{ :mysql2 }` fallback block?

Reply:
I assume we want to be able to run `bundle exec rake test` without having to set `TEST_ADAPTER_MYSQL`.